static/js/server_events_dispatch.js

```diff
@@ -214,7 +214,7 @@ export function dispatch_normal_event(event) {
                 private_message_policy: noop,
                 send_welcome_emails: noop,
                 message_content_allowed_in_email_notifications: noop,
-                realm_enable_spectator_access: noop,
+                enable_spectator_access: noop,
```

Comment:
Seeing that this didn't fail any tests makes me think we might want to harden this code against bugs, by throwing an error if the function isn't in this object.  (It seems like the design intent is for every realm property to be here).

Reply:
OK, and a final approach would be something involving `typescript`.  @priyank-p FYI in case you have ideas there.